metadata-builders/src/main/java/com/linkedin/metadata/builders/search/DataProcessIndexBuilder.java

```diff
@@ -62,23 +61,19 @@ private DataProcessDocument getDocumentToUpdateFromAspect(@Nonnull DataProcessUr
         return dataProcessDocument;
     }
 
-    @Nonnull
-    private DataProcessDocument getDocumentToUpdateFromAspect(@Nonnull DataProcessUrn urn, @Nonnull Status status) {
```

Comment:
Why is this removed?

Reply:
`Status` is not part of `DataProcessAspect` https://github.com/linkedin/datahub/blob/master/metadata-models/src/main/pegasus/com/linkedin/metadata/aspect/DataProcessAspect.pdl. This part of code is not being used.